### PR TITLE
Make reward application rows distinct with empty value

### PR DESCRIPTION
**Pull request by Devorein**
### WHY

image

image

### CODE REVIEW

File: components/common/BoardEditor/focalboard/src/components/table/tableRow.tsx [hunk 1]
```diff
@@ -258,6 +260,7 @@ function TableRow(props: Props) {
               displayType='table'
               columnRef={columnRef}
               wrapColumn={activeView.fields.columnWrappedIds?.includes(template.id)}
+              subRowsEmptyValueContent={subRowsEmptyValueContent}
```
Comment (motechFR) [inline components/common/BoardEditor/focalboard/src/components/table/tableRow.tsx:263]: I would add a comment for future maintainers about the reason for this field

**Review by motechFR — APPROVED** [review]

File: components/rewards/RewardsPage.tsx [hunk 2]
```diff
@@ -241,6 +241,7 @@ export function RewardsPage({ title }: { title: string }) {
                     onCardClicked={() => {}}
                     onDeleteCard={onDelete}
                     expandSubRowsOnLoad
+                    subRowsEmptyValueContent='--'
```
Comment (motechFR) [inline components/rewards/RewardsPage.tsx:244]: Suggestion: Would a single dash look better?

File: components/common/BoardEditor/components/kanban/KanbanGroupColumn.tsx [hunk 3]
```diff
@@ -58,6 +60,7 @@ export function KanbanGroupColumn({
           onDrop={onDropToCard}
           showCard={showCard}
           isManualSort={isManualSort}
+          hideLinkedBounty={hideLinkedBounty}
```
Comment (motechFR) [inline components/common/BoardEditor/components/kanban/KanbanGroupColumn.tsx:63]: What does this field do?

File: components/rewards/RewardsPage.tsx (hunk 2 above)
Comment (Devorein) [reply components/rewards/RewardsPage.tsx:244]: Chris specifically mentioned to use `--` in the product card
> Use "--" for all columns that are currently empty on applications

**Pull request merged**